### PR TITLE
Fix webassets version to 0.11.1

### PR DESCRIPTION
**Pull request by zazasa**
Fix webassets version due to problem parsing scss files

### CODE REVIEW

**Comment by jacquerie** [comment]
Had to do the same on INSPIRE: https://github.com/inspirehep/inspire-next/commit/f00e62b36840fe75a86e6bf0a22f960208378b23. What is causing this problem?

**Comment by andrewm89** [comment]
👌

File: setup.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
     'Flask-Assets>=0.11,<0.12',  # 0.12 will install Flask command 'assets'
     'Flask-Collect==1.2.2',
     'node-semver>=0.1.1',
-    'webassets>=0.11.1'
+    'webassets==0.11.1'
```
Comment (lnielsen) [inline setup.py]: Can you use `` 'webassets>=0.11.1,<0.12',`` instead like @jacquerie did on Inspire.

**Comment by lnielsen** [comment]
Did you check what is causing the problem in webassets? It might be an easy fix - in general we prefer fixing the issue and bumping the minimal requirement so that we stay on the latest released packages. Maximum requirements tend to stay for a very long time in the code (e.g. Invenio-OAuth2Server was for a long time pinned).

**Comment by zazasa** [comment]
I can agree in general, but it took long time to find the problem, and it requires further time to investigate deeper. 
Anyway I think that the fix will come together with the issue https://github.com/inveniosoftware/invenio-assets/issues/36 .

**Comment by jirikuncar** [comment]
We should try with `Flask-Assets>=0.12`.

File: setup.py [hunk 2]
```diff
@@ -64,7 +64,7 @@
     'Flask-Assets>=0.11,<0.12',  # 0.12 will install Flask command 'assets'
     'Flask-Collect==1.2.2',
     'node-semver>=0.1.1',
-    'webassets>=0.11.1'
+    'webassets>=0.11.1,<0.12'
```
Comment (jirikuncar) [inline setup.py]: Can you add a comma at the end of list. Thanks!

--
I know it was previously missing.

**Comment by jirikuncar** [comment]
@zazasa please check #38 and #36 that are related.

**Pull request closed without merging**